week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,45 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// Add comments to explain what this function does. You're meant to use Google!!!..
+
+// returns a Number value with positive sign, greater than or equal to 0 but less than 1 , chosen randomly or pseudo-randomly with approximately uniform distribution over that range and multiply it to 10
+
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+//The concat() function concatenates the string arguments to the calling string and returns a new string.Here we concated w1 to W2 and return a new value.
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
-    // Look at the test case below to understand what to expect in return 
+  // Write the body of this function to concatenate three words together
+  // Look at the test case below to understand what to expect in return
+  return firstWord + " " + secondWord + " " + thirdWord;
```

Comment:
Here it is wrong Bekoo. You have been asked to use concate method. could you fix that.